collector/drm_linux.go

```diff
@@ -99,11 +99,7 @@ func NewDrmCollector(logger log.Logger) (Collector, error) {
 }
 
 func (c *drmCollector) Update(ch chan<- prometheus.Metric) error {
-	if err := c.updateAMDCards(ch); err != nil {
-		return err
-	}
-
-	return nil
+	return c.updateAMDCards(ch)
 }
 
 func (c *drmCollector) updateAMDCards(ch chan<- prometheus.Metric) error {
```

Comment:
How about we just drop this function since the Update is is now just a wrapper.

Reply:
I'd say we keep this as hint for further contributors that they shall add an updateNVIDIACards or whatever function in